train.py

```diff
@@ -473,6 +485,7 @@ def main(args):
 				0: 'B', 2: 't'
 			})
 		)
+
 		onnxruntime_session = onnxruntime.InferenceSession(args.onnx)
```

Comment:
please replace this code by using the wrapper

Reply:
done